config/private/plugins.js

```diff
@@ -57,6 +57,9 @@ export default {
         ].filter(x => x != null),
 
         plugins: [
+          // this is used by mobx, and has to list at first
+          // https://mobxjs.github.io/mobx/best/decorators.html
+          'transform-decorators-legacy',
```

Comment:
Hmmm, some initial feedback from me here.  Personally I am against the use of any syntax that hasn't made it to stage 3 of the TC39 process.  It is a risk to include anything lower as things can be dropped or changed.  That's why we now have this "legacy decorators" plugin. :)

Would you be happy to update your branch to not use decorators?  Does `mobx` provide enough higher order functions as an alternative?